### PR TITLE
fix: date/time format for online edition

### LINKED ISSUE #482: [BUG] Event not marked as ended
### Describe the bug.

In First event due to wrong date time format it is not marking that event as "ENDED"

### Expected behavior

after passing of given date it should mark the event as ended

### Screenshots

Screenshot from 2024-11-29 22-29-22

### How to Reproduce

go on 1st event on home page to see

To FIX :
    change ( 30th October 2024 --> october 30 2024 )

### 🥦 Browser

None

### 👀 Have you checked for similar open issues?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md)

### Are you willing to work on this issue ?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by anmolyadav-dev**
This PR Fixes #482 

Screenshot from 2024-11-29 22-55-10

### CODE REVIEW

**Review by ashmit-coder — APPROVED** [review]
LGTM!

File: config/city-lists.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	{
 		"name": "Online",
 		"country": " Edition",
-		"date": "30th October 2024",
+		"date": "October 30 2024",
```
Comment (ashmit-coder) [inline config/city-lists.json:5]: add a comma before 2024, like in paris and london

**Review by ashmit-coder — CHANGES_REQUESTED** [review]
Add the comma for `helsinki` date as well

**Comment by ashmit-coder** [comment]
cc: @anmolyadav-dev

File: config/city-lists.json (hunk 1 above)
Comment (anmolyadav-dev) [reply config/city-lists.json:5]: yeah sure! doing right away

**Comment by ashmit-coder** [comment]
hey  @anmolyadav-dev could you edit the cypress test so that the url for subcribe button matches the regex for `https://www.asyncapi.com/en/newsletter`

that is the reason your pr is failing on the tests

**Review by ashmit-coder — APPROVED** [review]
LGTM!

**Review by AceTheCreator — APPROVED** [review]
LGTM!

**Pull request merged**